releasenotes/notes/0.24/update-state-visualization-6836bd53e3a24891.yaml

```diff
@@ -1,23 +1,25 @@
 ---
-features:
+features_visualization:
   - |
-    The ``figsize`` argument of
-    :obj:`~qiskit.visualization.plot_bloch_multivector` can now be used to
-    set a size for individual Bloch ball sub-plots: if there are ``num``
-    qubits and ``(w, h)`` is passed to ``figsize``, then the overall figure
-    width is set to ``num*w``, while the overall height is set to ``h``.
+    Added a new keyword argument, ``figsize``, to the
+    :func:`~qiskit.visualization.plot_bloch_multivector` function. This
+    argument can be used to set a size for individual Bloch sphere sub-plots.
+    For example, if there are :math:`n` qubits and ``figsize`` is set to
+    ``(w, h)``, then the overall figure width is set to :math:`n*w`, while the
```

Comment:
The star usually renders weirdly as multiplication, maybe it's better to use the multiplication dot 
```suggestion
    ``(w, h)``, then the overall figure width is set to :math:`n \cdot w`, while the
```